parts/k8s/kubernetesconfigs.sh

```diff
@@ -117,6 +126,8 @@ ensureRPC() {
 runAptDaily() {
     wait_for_apt_locks
     /usr/lib/apt/apt.systemd.daily
+    wait_for_apt_locks
+    holdWALinuxAgent "unhold"
```

Comment:
`hold "unhold"` is bit unusual, but naming is hard.

Reply:
suggestion: `aptmarkWALinuxAgent "hold"/"unhold"`

Reply:
+1 on Stephane's proposed naming.

Reply:
for a follow-up PR